spec/Records.tex

```diff
@@ -186,23 +185,18 @@ \subsection{Record Initialization}
 is initialized through a call to the record's default initializer,
 passing no arguments.  The default initializer for a record is defined in
 the same way as the default initializer for a class
-(\rsec{Default_Initialization}).
+(\rsec{The_Compiler_Generated_Initializer}).
 
-To construct a record as an expression,
+To create a record as an expression,
 i.e. without binding it to a variable, the \chpl{new} operator is
 required.  In this case, storage is allocated and reclaimed as for a record
 variable declaration (\rsec{Record_Storage}), except that the temporary record
 goes out of scope at the end of the enclosing statement.
```

Comment:
That's neither what's implemented nor what we rely on today for `owned`. Can it say "the temporary record goes out of scope at the end of the enclosing block"?